test/functional/desired.js

```diff
@@ -16,8 +17,13 @@ const APIDEMO_CAPS = Object.assign({}, GENERIC_CAPS, {
   app: apidemosApp,
 });
 
+const COMPOSE_CAPS = Object.assign({}, GENERIC_CAPS, {
```

Comment:
👍

Reply:
It took me a hard time to figure out that the test file name did not have 'e2e' and it was executing as part of unit tests. Where is this logic configured? is it some convention?

Reply:
I believe this logic should live somewhere in https://github.com/appium/appium-gulp-plugins or https://github.com/appium/appium-test-support